internal/service/instance_test.go

```diff
@@ -32,3 +38,24 @@ func TestInstanceService_GetInstances(t *testing.T) {
 	instanceService.UpdateInstances(testInstances)
 	assert.Equal(t, testInstances, instanceService.GetInstances())
 }
+
+func createTestIds() (uuid.UUID, uuid.UUID, error) {
+	tenantId, err := uuid.Parse("7332d596-d2e6-4d1e-9e75-70f91ef9bd0e")
+	if err != nil {
+		fmt.Printf("Error creating tenantId: %v", err)
+	}
+
+	instanceId, err := uuid.Parse("aecea348-62c1-4e3d-b848-6d6cdeb1cb9c")
+	if err != nil {
+		fmt.Printf("Error creating instanceId: %v", err)
+	}
+
+	return tenantId, instanceId, err
+}
+
+func createProtoTime(timeString string) (*timestamppb.Timestamp, error) {
+	time, err := time.Parse(time.RFC3339, timeString)
+	protoTime := timestamppb.New(time)
+
+	return protoTime, err
+}
```

Comment:
Are these used anywhere?

Reply:
Nope, forgot to remove them when we changed where everything was